pkg/networkplugin-syncer/handlers/ovn/infrastructure.go

```diff
@@ -72,6 +72,10 @@ func (ovn *SyncHandler) ensureSubmarinerRouter() error {
 		[]string{submarinerDownstreamNET}, nil,
 	)
 
+	if err != nil {
+		return errors.Wrapf(err, "error link for port %q", submarinerDownstreamRPort)
+	}
```

Comment:
The `err` referenced here pertains to a prior operation. Is this intended be above L63? Or is this intended to check the error return from L68 (which is currently ignored)?.

Reply:
oops!, I was trying to add error handling to the linkCmd generation, I'll fix that.

Reply:
Done.